### DISCUSSION TITLE
embedding a nextjs/react based widget in existing webpage?

### DISCUSSION THREAD

**Discussion by aktxyz** [vercel/next.js#25320] [Help]
Is there a clean way to embed a widget on an existing page? something like

```
<html>
<body>
existing stuff

<div id="nextWidgetGoesHere"></div>

<script src="https://somesite.com/mynextjswidgetapp"></script>

</body>
</html>
```

**Comment by scottschindler** [comment]
What if you used an HTML iframe that points to your next.js site?

Link: https://www.w3schools.com/html/html_iframe.asp

**Reply by aktxyz** [reply]
yeah ... hoping to avoid the iframe route

this is a very good example ... and will fiddle with this soon and hopefully munge it into next

was hoping to find a nextjs example since I have been using nextjs for probably 5 years and can't imagine not using it

https://javascriptpros.com/creating-react-widgets-embedded-anywhere/
